qcodes/instrument_drivers/tektronix/Keithley_6500.py

```diff
@@ -121,17 +126,32 @@ def __init__(self, name, address, reset_device=False, **kwargs):
                            vals=MultiType(Ints(min_value=1, max_value=9999),
                                           Enum('inf', 'default', 'minimum', 'maximum')))
 
-        self.add_parameter('trigger_delay',
-                           get_parser=float,
-                           get_cmd='TRIG:DEL?',
-                           set_cmd='TRIG:DEL {}',
-                           unit='s', vals=Numbers(min_value=0, max_value=999999.999))
-
-        self.add_parameter('trigger_source',
-                           get_cmd='TRIG:SOUR?',
-                           set_cmd='TRIG:SOUR {}',
-                           val_mapping={'immediate': 'NONE', 'timer': 'TIM', 'manual': 'MAN',
-                                        'bus': 'BUS', 'external': 'EXT'})
+        for trigger in range(1, 5):
+            self.add_parameter('trigger%i_delay' % trigger,
+                               docstring='Set and read trigger delay for '
+                                         'timer %i.' % trigger,
+                               get_parser=float,
+                               get_cmd='TRIG:TIM%i:DEL?' % trigger,
+                               set_cmd='TRIG:TIM%i:DEL {}' % trigger,
+                               unit='s', vals=Numbers(min_value=0,
+                                                      max_value=999999.999))
+
+            self.add_parameter('trigger%i_source' % trigger,
+                               docstring='Set, and read, the event that sets'
```

Comment:
this docstring is confusing: "set and read the event"?

Reply:
It is confusing, fixed.